builds/msvc/vs2013/libbitcoin-explorer/libbitcoin-explorer.vcxproj

```diff
@@ -79,6 +79,7 @@
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer.hpp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\prop_tree.ipp" />
     <ClInclude Include="..\..\..\..\include\bitcoin\explorer\impl\utility.ipp" />
+    <ClInclude Include="..\..\..\..\include\bitcoin\explorer\async_client.hpp" />
```

Comment:
@evoskuil Correct, the command line that you specified runs properly (positional args work).

The async_client is in the template, so it appears in the files when I run generate.sh. I'll remove it.

Not sure what you mean about the stub subfolder though.  I removed stub="true" and re-generated, but the layout of files has not changed.